src/js/components/Notification/stories/Toast.js

```diff
@@ -9,7 +9,7 @@ import { addToast } from '..';
 
 const Toast = () => {
   const addToastDefault = () => {
-    addToast({ msg: 'I am default toast' });
+    addToast({ msg: 'I am default toast', autoClose: false });
```

Comment:
do not add the autoclose option to existing toast. add a separate story for that